temporal-opentracing/src/main/java/io/temporal/opentracing/internal/ActionTypeAndNameSpanBuilderProvider.java

```diff
@@ -88,12 +88,16 @@ protected Map<String, String> getSpanTags(SpanCreationContext context) {
       case RUN_WORKFLOW:
       case START_ACTIVITY:
       case RUN_ACTIVITY:
+      case SIGNAL_WORKFLOW:
+      case UPDATE_WORKFLOW:
         String runId = context.getRunId();
         Preconditions.checkNotNull(
             runId, "runId is expected to be not null for span operation type %s", operationType);
         return ImmutableMap.of(
             StandardTagNames.WORKFLOW_ID, context.getWorkflowId(),
             StandardTagNames.RUN_ID, context.getRunId());
+      case QUERY_WORKFLOW:
+        return ImmutableMap.of();
```

Comment:
In our other SDKs, query spans also have workflow ID and run ID.

Reply:
The problem here is that handleQuery is not executed within a workflow thread, leading to non-deterministic errors.

Reply:
Yes

Reply:
I did change it for `QUERY_WORKFLOW` case but it is not possible for `HANDLE_QUERY` because it is not running inside a workflow thread.

Reply:
@cretz He is  saying he cannot set them the same way as `HANDLE_SIGNAL` because in the Java SDK queries cannot call ` Workflow.getInfo()`

Reply:
Can store as state in the interceptor probably, but I guess the question is, should we get the workflow info in here properly or is this good enough for now and we change later?

Reply:
You can't currently get the workflow info here "properly" and it is a large change to enable it

Reply:
I'm not sure that a span without these tags is ok, but I'll let y'all decide

Reply:
Well the options are either no span or a span without those tags. I have no preference.

Regardless of which we go with we will need to open an issue to address this when we fix `getInfo` in query handlers , but I don't think that should block this PR.

Reply:
Concur, will let you make the call on tagless span vs no span. I don't have a super-strong opinion here.

Reply:
I believe it would be better to have a tagless span since the span itself is useful for tracing.

Reply:
Works for me